source/utilities.tex

```diff
@@ -16959,6 +17011,49 @@
 supported\iref{basic.align}.
 
 \indexlibrary{\idxcode{common_type}}%
+\pnum
+Let:
+\begin{itemize}
+\item \tcode{CREF(A)} be \tcode{add_lvalue_reference_t<const remove_reference_t<A>{}>},
```

Comment:
We usually do \placeholder here (cf. INVOKE), because CREF(A) is not valid C++ source code; it's a funny abstract-level meta-type-function.